cmd/otel-allocator/prehooktargetfilter/no_op_filter.go

```diff
@@ -0,0 +1,40 @@
+package prehooktargetfilter
```

Comment:
Can you run golangci-lint and fix any lint issues in these new files? They need license comment blocks.

Reply:
Thanks for pointing this out! Added the copyright to the top of the new files and not seeing any linting errors.